### PR TITLE
built image component

### PR DESCRIPTION
**Pull request by Rockyraven**
Please review my PR request

### CODE REVIEW

File: component/image/image.html [hunk 1]
```diff
@@ -1,58 +1,70 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Image-component</title>
-    <link rel="stylesheet" href="image.css">
+    <title>BOOSTWEB-UI | Image</title>
```
Comment (zabihshaik) [inline component/image/image.html:8]: Better to use meta tag with your name in the content

**Review by zabihshaik — COMMENTED** [review]
Everything looks good. Just check the comment please.

**Review by sagargse — APPROVED** [review]
looks good to me

File: component/image/image.html (hunk 1 above)
Comment (Rockyraven) [reply component/image/image.html:8]: is it complusory

Comment (zabihshaik) [reply component/image/image.html:8]: Better to use for SEO. I have seen the MAs commenting about it in their Reviews.

Comment (zabihshaik) [reply component/image/image.html:8]: Not compulsory I guess. But, better to have

**Pull request merged**